core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/version/JavaVMVersionToolkit.java

```diff
@@ -91,8 +91,8 @@ private static boolean isNumber(String string) {
 	 *
 	 * @param vmName
 	 *            the JVM name to check.
-	 * @return {@code true} of it is a JRockit, {@code false} if it isn't or if was not possible to
-	 *         tell.
+	 * @return {@code true} of it is a JRockit, {@code false} if it isn't or if it was not possible
```

Comment:
Haha, as you touch it, you should also fix this typo: "of it is" -> "if it is" 😄

Reply:
Boo, I should have taken a bit of a deeper read there .. thanks for checking again!